src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/GenerateOrderCallbackHandlerTest.java

```diff
@@ -80,10 +90,23 @@ public void setUp() {
             docAssemblyClient,
             authTokenGenerator,
             jsonMapper,
-            docAssemblyTemplateBodyMapper
-        );
+            docAssemblyTemplateBodyMapper,
+            caseDetailsConverter);
         ReflectionTestUtils.setField(generateOrderCallbackHandler, "templateId", "testTemplateId");
         ccdCase = SampleData.getCCDCitizenCase(Collections.emptyList());
+        Claim claim =
+            SampleClaim.builder()
+                .withResponse(
+                    FullDefenceResponse.builder()
```

Comment:
SampleResponse may have this FullDefenceResponse provider. If not pls add it to there so others can use too. Thanks